Automattic/wp-calypso
# Checkout: Disable Paypal payments in the desktop app

Fixes: https://github.com/Automattic/wp-calypso/issues/3410

With this change, Paypal payments are disabled in the desktop application

Before:
![image](https://cloud.githubusercontent.com/assets/230230/13223638/5f014e06-d985-11e5-9e49-4f1cdef621d0.png)


After:
![image](https://cloud.githubusercontent.com/assets/230230/13223601/3eb0ef94-d985-11e5-8a9e-77c4c73fbca0.png)


#### Testing Instructions
- Use an account without free credits.
- In wp-desktop, move to the `calypso` folder and run `git fetch origin && git checkout fix/disable-paypal-desktop`
- Move back to wp-desktop's root and run `make run`
- In app, try to purchase a plan (or anything else), you should land on the checkout page
- Check that `or use PayPal` is not available anymore on the bottom right of the payment form.

#### Reviews
